administrator/language/en-GB/plg_schemaorg_blogposting.ini

```
@@ -0,0 +1,26 @@
+; Joomla! Project
+; (C) 2023 Open Source Matters, Inc. <https://www.joomla.org>
+; License GNU General Public License version 2 or later; see LICENSE.txt
+; Note : All ini files need to be saved as UTF-8
+
+PLG_SCHEMAORG_BLOGPOSTING="Schema.org - Blogposting"
+PLG_SCHEMAORG_BLOGPOSTING_DESCRIPTION_LABEL="A blog post <a href=\"https://schema.org/BlogPosting\" target=\"_blank\">More information</a>"

```

please add 
```suggestion
PLG_SCHEMAORG_BLOGPOSTING_DESCRIPTION_LABEL="A blog post <a href=\"https://schema.org/BlogPosting\" target=\"_blank\"  rel=\"noopener noreferrer\">More information</a>"

```


This is for security purposes. Sorry your mentors didnt inform you of this